src/app/dashboard-widgets/analytical-report-widget/analytical-report-widget.component.ts

```diff
@@ -118,68 +114,10 @@ export class AnalyticalReportWidgetComponent implements OnInit, OnDestroy {
   selectedBuild(): void {
     let build: Build = this.currentBuild;
     this.showLoader();
+    this.stackUrl = '';
     if (build) {
       let url: string = build.annotations['fabric8.io/bayesian.analysisUrl'];
```

Comment:
This intermediate variable is unnecessary - you could just have this.stackUrl = build.annotations...